sdk/schemaregistry/Azure.Data.SchemaRegistry/samples/Sample01_HelloWorld.md

```diff
@@ -0,0 +1,124 @@
+# Registering and retrieving Avro, Json, and custom schemas (preview)
+
+The following shows examples of how to use the basic functionality of the `SchemaRegistryClient` with each of the available schema format types. All of the client methods can be used with each format by passing in the proper 'SchemaFormat' value when needed.
+
+Each Event Hubs namespace can only accept one format of schemas.
```

Comment:
Hmm is this actually true? If so, it would explain why we needed separate endpoints for each format, but I find this to be a strange restriction.

Reply:
I guess the real restriction is actually on the groups, but since only one group is allowed per namespace it's kind of the same thing, but I will change this to be groups to align better.

Reply:
Ah, that is unfortunate. I guess we will need to stick to creating the namespaces in test-resources.json then.